### PR TITLE
Empty else block inspections

### PR DESCRIPTION
**Pull request by IvenBach**
Contains inspections that should bring attention to else blocks. Let me know if there's something I missed or that doesn't look right. Any quick fix is beyond what I can understand right now. Should I join `EmptyIfBlockInspection` and `EmptyElseBlockInspection` to be a new `EmptyConditionBlockInspection`?

### CODE REVIEW

**Comment by Vogel612** [comment]
The simplest quickfix would be "Remove Else block", which boils down to rewriting the Else token in the relevant condition into nothingness :)

**Comment by MDoerner** [comment]
So far, you do not seem to handle empty `singleLineElseStatement`s. These have a special syntax where you can either use one or multiple `sameLineStatement`s separated by statement separators or a label as an implicit goto statement. For more details, have a look at the grammar.

**Comment by ThunderFrame** [comment]
Inline If/Else can have nested inline If/Else statements:

```vb
  If False Then If True Then Beep Else Beep Else If False Then Else If False Then Else
```

**Comment by IvenBach** [comment]
@ThunderFrame Why would anyone do such a thing as that? That's asking for problems.

**Comment by retailcoder** [comment]
@IvenBach we've come to accept that there's completely legal VBA code out there that you and I wouldn't write or maintain - put it this way: if you're cleaning up messy code, you don't want your ducky to let you down ;-)

File: RubberduckTests/Inspections/EmptyElseBlockInspectionTests.cs [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+            var inspector = InspectionsHelper.GetInspector(inspection);
+            var actualResults = inspector.FindIssuesAsync(state, CancellationToken.None).Result;
+            const int expectedCount = 1;
+
+            Assert.AreEqual(expectedCount, actualResults.Count());
+        }
+
+        [TestMethod]
+        [TestCategory("Inspections")]
+        public void EmptyElseBlock_HasVariableDeclaration()
```
Comment (retailcoder) [inline RubberduckTests/Inspections/EmptyElseBlockInspectionTests.cs]: Same as https://github.com/rubberduck-vba/Rubberduck/pull/3067/files#diff-a049b39cbf42b1ae566e5d7e0047062eR200

**Comment by IvenBach** [comment]
Changes to address [issue #3037] and  [issue #3031] inadvertently coupled to this pull request. I'll need some help creating a separate branch to do as their own pull request. Bear with me learning GitHub.

File: RetailCoder.VBE/UI/FindSymbol/FindSymbolControl.xaml.cs [hunk 2]
```diff
@@ -37,7 +37,8 @@ private void CommandBinding_OnCanExecute(object sender, CanExecuteRoutedEventArg
 
         private void UIElement_OnPreviewKeyDown(object sender, KeyEventArgs e)
         {
-            if (e.Key == Key.Enter && ViewModel.CanExecute())
+            //if (e.Key == Key.Enter && ViewModel.CanExecute())
```
Comment (rubberduck203) [inline RetailCoder.VBE/UI/FindSymbol/FindSymbolControl.xaml.cs]: Commented out code should be deleted please.

**Comment by IvenBach** [comment]
Just pulled latest build into my local. There were tons of errors. I'm going to close this PR for now and submit a new one once I get all the issues ironed out.

**Pull request closed without merging**